src/audio/google/google_rtc_audio_processing.c

```diff
@@ -607,14 +609,24 @@ static int google_rtc_audio_processing_prepare(struct processing_module *mod,
 	if (mic_fmt != bits_fmt(CONFIG_GOOGLE_RTC_AUDIO_PROCESSING_MIC_BITS)) {
 		comp_err(dev, "Mic stream must be %d bit samples\n",
 			 CONFIG_GOOGLE_RTC_AUDIO_PROCESSING_MIC_BITS);
-		ret = -EINVAL;
+
+		// FIXME: squash for now, the streams lie.  See below.
+		//ret = -EINVAL;
 	}
 
 	if (mic_fmt != out_fmt) {
 		comp_err(dev, "Mismatched in/out frame format");
 		ret = -EINVAL;
 	}
 
+	// FIXME: the streams have a bad format on MTL, so we can't
+	// use this API.  Compute by hand.
+	//
+	//cd->ref_frame_bytes = source_get_frame_bytes(sources[cd->aec_reference_source]);
+	//cd->out_frame_bytes = sink_get_frame_bytes(sinks[0]);
```

Comment:
You probably made it "incoform" on purpose, but maybe putting this in a normal C style multi-line comment would be enough

Reply:
Yeah, it was to make this eye catching to be sure it gets fixed. If there's concern about merging the workaround like this we can split it out for merge on the device development branch (heh, excuse me, "stable drop").